Project/src/main/resources/templates/upload.html

```diff
@@ -56,7 +56,7 @@
                 <tr th:id="'article' + ${article.id}" scope="row" th:each="article: ${articles}">
                     <td th:text="${article.fileName}"></td>
                     <td th:text="${article.date}"></td>
-                    <td th:text="${article.review}"></td>
+                    <td th:text="${article.getReviewer()}"></td>
```

Comment:
I think rather than changing the toString, you should do this if you can
```suggestion
                    <td th:text="${article.getReviewer().getUsername()}"></td>
```